_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -478,19 +478,31 @@ def post(self, url: str, data: dict = {}):
 class FrontstageTasks(TaskSet, Mixins):
 
     def on_start(self):
-        self.sign_in()
+        requests_filepath = "requests.json"
 
-    def sign_in(self):
-        response = self.get(url="/sign-in", expected_response_text="Sign in")
-        self.csrf_token = _capture_csrf_token(response.content.decode('utf8'))
-
-        response = self.post("/sign-in", data=_generate_random_respondent())
-        self.auth_cookie = response.cookies['authorization']
+        with open(requests_filepath, encoding='utf-8') as requests_file:
+            requests_json = json.load(requests_file)
+            self.requests = requests_json["requests"]
 
     @task
-    def todo(self):
-        self.get("/surveys/todo", expected_response_text="Click on the survey name to complete your questionnaire")
+    def perform_requests(self):
+        for request in self.requests:
+            request_url = request['url']
+
+            if request["method"] == "GET":
+                request_response = request['expected_response_text']
+                response = self.get(request_url, request_response)
```

Comment:
I named var it as it not usual in a get, when that happens it's good practise to help the developer, however if the var was named more appropriate it might be ok. One to try and see how it looks with different combinations

Reply:
These request commands in the for-loop are no longer assigned to a variable.

Reply:
Its slightly confusing having the `get()` and `post()` methods in the Mixins also do the assertions, and then return the HTTP response back here. But Python doesn't seem to care these methods return a value that's not assigned to anything. Again this is good enough and we  and change it as/when required.